python/lsst/pipe/base/pipeline.py

```diff
@@ -484,6 +526,9 @@ def _addConfigImpl(self, label: str, newConfig: pipelineIR.ConfigIR):
     def toFile(self, filename: str):
         self._pipelineIR.to_file(filename)
 
+    def toUri(self, uri: str):
+        self._pipelineIR.to_uri(uri)
+
```

Comment:
Pre-existing line below: `typing.Generator` requires three type parameters, not just one (the other two are `None` in this case).  FWIW, this is one of the reasons I  prefer `Iterator` when the user shouldn't care how the lazy iterator is implemented.

Reply:
The typing "guidelines" I have read have been "Be liberal in what you accept, be specific with what you return" so I try to adhere to that as much as possible. It does not hurt to have more info about what is being returned, but it is a pain to get more specific later (especially in a case where it suddenly does matter)